apps/system/utils/fscmd.c

```diff
@@ -282,6 +282,10 @@ static int tash_cat(int argc, char **args)
 		FSCMD_OUTPUT(MISSING_ARGS FSCMD_CAT_USAGE, args[0]);
 		return ERROR;
 	} else if (argc == 2) {
+		if (!strncmp(args[1], "--help", 6)){
```

Comment:
Including null terminator, the length of strncmp might be 7.

Reply:
fixed